clusterloader2/util/unmarshaling.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+var codecs = serializer.NewCodecFactory(runtime.NewScheme())
```

Comment:
I started wondering if that will work in our case. it depends on the compiled in (thus initialized) api-groups.
This may or may not work in our purpose - so I will hold-on with lgtm-ing this one.

Reply:
Probably there will be import to legacyschema added (and all apis). I just realized that initialized scheme is also required for validation PR.

Reply:
Validation? You're not validating objects so far..

Reply:
I meant pull request #142 - validation for test configs. I expect that if we want to have validation for ObjectType, we have to have scheme initialized.
It is just another argument for adding initialization for api groups...

Reply:
It isn't obvious to me we want to do that - this means that adding tests for newly added object APi requires rebuliding cluster loader. It's even more important when we start thinking about testing stuff for validation before it's merged.

Maybe it's actually enough to read it to runtime.Unstructured, where we don't really need to know the schema.

And for detecting the type of the object - parse it to e.g. metav1.PartialObjectMetadata

Reply:
So I would say - let's for now remove it completely, and merge this PR just with parsing config merged.

Reply:
Ok. I'm moving ReadTemplate method to different PR.